worker-message-prioritization-distribution-container/pom.xml

```diff
@@ -0,0 +1,233 @@
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                    </execution>
+
+                </executions>
+
+                <configuration>
+                    <autoPull>on</autoPull>
+                    <containerNamePattern>%a-%t</containerNamePattern>
+                    <verbose>true</verbose>
+                    <images>
+
+                        <!-- Run the RabbitMQ image -->
+                        <image>
+                            <alias>rabbitmq</alias>
+                            <name>${dockerHubPublic}/library/rabbitmq:3-management</name>
+                            <run>
+                                <ports>
+                                    <port>${rabbitmq.ctrl.port}:15672</port>
+                                    <port>${rabbitmq.node.port}:5672</port>
+                                </ports>
+                                <wait>
+                                    <log>Server startup complete</log>
+                                    <http>
+                                        <url>http://${docker.host.address}:${rabbitmq.ctrl.port}</url>
+                                    </http>
+                                    <time>120000</time>
+                                    <shutdown>500</shutdown>
+                                </wait>
+                                <log>
+                                    <enabled>true</enabled>
+                                </log>
+                            </run>
+                        </image>
+
+                        <image>
+                            <name>${dockerWorkerFrameworkOrg}worker-message-prioritization-distribution${dockerProjectVersion}</name>
+                            <!-- Build the image for this module -->
+                            <build>
+                                <from>${dockerHubPublic}/cafapi/opensuse-jre11:3</from>
+                                <labels>
+                                    <Build.Number>${project.version}</Build.Number>
+                                    <Build.Date>${maven.build.timestamp}</Build.Date>
+                                    <Git.Repo>https://github.com/WorkerFramework/worker-message-prioritization</Git.Repo>
+                                    <Git.Branch>${git.branch}</Git.Branch>
+                                    <Git.Commit>${git.revision}</Git.Commit>
+                                </labels>
+                                <healthCheck>
+                                    <!--
+                                        Check if the Java process is running. This also indirectly verifies that the connection to
+                                        RabbitMQ is ok, since the Java process will exit if it cannot connect to RabbitMQ.
+                                    -->
+                                    <cmd>/maven/healthcheck/healthcheck.sh || exit 1</cmd>
```

Comment:
Although healthcheck.sh contains only a small command, it needs to be in a script to work. When I tried to put it inline in the pom.xml here:

```
<cmd>pgrep -f "java.*worker-message-prioritization-distribution.*jar" > /dev/null || exit 1</cmd>
```

the healthcheck always reported healthy, even when it should not have.